doc/faq.rst

```diff
@@ -34,6 +33,14 @@ a full error report with all the typical information (OS, version,
 console-output, minimum working example, ...). This makes it a lot easier to
 reproduce the error and locate the problem.
 
+.. rubric:: What does the *deterministic* flag in a scenario mean?
+
+If the *deterministic* flag is set to **False** the target algorithm is assumed to be non-deterministic.
+To evaluate a configuration of a non-deterministic algorithm, multiple evaluations with different seeds are necessary
+to determine the performance of that configuration on one instance.
+Deterministic algorithms don't depend on seeds, thus requiring only one evaluation of a configuration on an instance
```

Comment:
please mention that the default seed is 0 ... nevertheless passed to the target algorithm